test/Models/CesiumSpec.ts

```diff
@@ -296,6 +296,120 @@ describeIfSupported("Cesium Model", function () {
       );
     });
   });
+
+  describe("Cesium Terrain Extra Tests", function () {
+    // declare different variables here, we need new instances to test
+    let terria2: Terria;
+    let terriaViewer2: TerriaViewer;
+    let container2: HTMLElement;
+    let cesium2: Cesium;
+
+    beforeEach(function () {
+      terria2 = new Terria({
+        baseUrl: "./"
+      });
+      terriaViewer2 = new TerriaViewer(
+        terria2,
+        computed(() => [])
+      );
+      container2 = document.createElement("div");
+      container2.id = "container2";
+      document.body.appendChild(container2);
+    });
+
+    afterEach(function () {
+      cesium2?.destroy();
+      document.body.removeChild(container2);
+    });
+
+    it("should thow an warning when cesiumIonAccessToken is invalid", async function () {
+      runInAction(() => {
+        terria2.workbench.removeAll();
```

Comment:
Is this necessary?

Reply:
No, not necessary, blindly copied from other tests pattern. I have removed now.